oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,399 @@
+and held accountable for its performance.
+
+The Open edX project, like many projects, calls this role
+"maintainer."  The notion of a project maintainer is typical of
+open-source projects and widely considered an best practice [#]_.
+
+The maintainer role is a leadership role that encompasses both
+technical and community management elements.  Both dimensions of the role are
+equally important.
+
+All, non-deprecated, components of the Open edX project must have one
+or more named maintainers.  When no such maintainers exists, this will
+be considered an urgent problem to solve.
+
+This OEP proposes a formal definition of the role of maintainer that
+will open to all qualified Open edX community members.  Maintainers
+will be responsible for the health of platform components, but not for
+their performance in any particular community member's production
+environment[#]_.
+
+Motivation
+==========
+
+With the change in stewardship from edX to The Center for Reimagining
+Learner, it is essential to have a well defined plan for maintaining
+the platform.
+
+In order to ensure that the project is sustainable for the long term,
+maintenance must be shared by the entire community.  The project must
+be robust enough that it can tolerate firms and contributors leaving
+the ecosystem.
+
+As a project we must acknowledge that maintenance is hard work, but
+work that must be done for the platform to be fit for use.  The best
+way of ensuring this is to have a named person or persons who are
+responsible for getting the work done, a measurable definition of
+"well-maintained," and feedback looks to ensure accountability and
+continuous improvement of our processes.
+
+Specification
+=============
+
+Definitions
+-----------
+
+The term **repository** will mean a repository on the GitHub social
+coding platform.
+
+The term **component** will typically mean the unit of software housed
+in a single GitHub repository.  Exceptions will be made in the case of
+edx-platform where there may be multiple distinguishable components in
+that single repository.  Components will typically, but not always, be
+hosted at the `openedx GitHub organization
+<https://github.com/openedx/https://github.com/openedx/>`_.
+Components will be "part of" or "critical to" the open-source
+platform.
+
+The term **maintainer** will mean a named person or persons
+responsible for the health of the component and the component's community.
+The remainder of this specification will be focused on the details of
+the role.
+
+The term **core contributor** will mean a named person participating
+in the `Core Contributor Program`_ as a coding core contributor and
+meeting the requirements of that program.
+
+The term **maintainer scrum-of-scrums** will mean the group of all
+maintainers who are responsible for flagging issues to project
+leadership and proposing process improvements for component
+maintenance.
+
+The term **Technical Oversight Committee** (TOC) will mean the group
```

Comment:
This is the only instance where TOC is mentioned. Why? And if no where else in doc, why do we need to define this organization?

Reply:
The TOC is ultimately the arbiter of last resort for governance matters.  I should make sure that that is perfectly clear here.

Reply:
These are good details to include. Since the TOC isn't referenced anywhere else, though, can you move this out of the definitions section?

Reply:
I think this point is a bit of a nit. It's important to mention that the TOC is the arbiter of last resort for governance matters.

Could we resolve this comment? If not, would you be able to float a specific suggestion?

Reply:
@sarina @e0d Honestly, I'm not sure how the TOC fits into this OEP, so I wouldn't know what to suggest. Can we just remove the definition?

If we're going to mention the TOC in an OEP, it may as well be OEP-1, since their governance-of-last-resort role applies to everything about Open edX, not just maintainership.

Reply:
Personally I do think this is worth mentioning; it's pertinent to the maintainers program to understand the governance structure, and we have a lot of OEPs with a lot of words - I don't think there's a problem with repeating this, especially as it is new -- people who may already have read OEP-1 may not have read a governance update.

What about something like

### Governance Structure

The Maintainers program will be run by the maintainers themselves, working together with input from the broader community. The Program Administrators (link to existing section at end) will serve as a backstop for any issues that arise, and the Technical Oversight Committee (move this section to the end & link to it) will be the arbiter of last resort for governance matters.

Reply:
I decided to leave the definition here, because it would be structurally a bit weird to have definitions at the top and bottom. But now there's that "Governance Structure" section at the end, so TOC is referenced.

Reply:
💯